src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -4132,65 +3927,10 @@ class AAttribute: Attribute
             End Using
         End Function
 
-        <WorkItem(362890, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=362890")>
-        <InlineData(CompletionImplementation.Legacy)>
-        <InlineData(CompletionImplementation.Modern, Skip:="https://github.com/dotnet/roslyn/issues/29110")>
-        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function TestFilteringAfterSimpleInvokeShowsAllItemsMatchingFilter(completionImplementation As CompletionImplementation) As Task
```

Comment:
❔ Is this moved or deleted?

Reply:
Was missed! Thank you!
Actually, I found that filter tests didn't properly work. 
Fixed them.

---
In reply to: [325725534](https://github.com/dotnet/roslyn/pull/38569#discussion_r325725534) [](ancestors = 325725534)